agent/src/issuers/mvrc.ts

```diff
@@ -0,0 +1,214 @@
+import { ClaimFormat, JwaSignatureAlgorithm } from '@credo-ts/core'
+import { OpenId4VciCredentialFormatProfile } from '@credo-ts/openid4vc'
+
+import { AGENT_HOST } from '../constants'
+import type {
+  CredentialConfigurationDisplay,
+  MdocConfiguration,
+  PlaygroundIssuerOptions,
+  SdJwtConfiguration,
+} from '../issuer'
+import type { StaticMdocSignInput, StaticSdJwtSignInput } from '../types'
+import { DateOnly, dateToSeconds, oneYearInMilliseconds, serverStartupTimeInMilliseconds } from '../utils/date'
+
+const mvrcDisplay = {
+  locale: 'en',
+  name: 'Vehicle Registration',
+  text_color: '#FFFFFF',
+  background_color: '#CD3401',
+  background_image: {
+    url: '',
+    uri: '',
```

Comment:
Is this correct? I empty uri's don't work well in the wallet. Can't we omit them?

Reply:
Removing causes errors in the playground, and it works fine in the wallet.